e2e/pages/home.page.js

```diff
@@ -0,0 +1,10 @@
+var HomePage = function () {
+    
+    this.greetingHeader = $('main.container h1');
```

Comment:
its a good practice to have specific attributes for such locators
for example, `data-e2e="layout::header__title"`